google-cloud-spanner/src/test/java/com/google/cloud/spanner/it/ITQueryTest.java

```diff
@@ -771,7 +767,7 @@ public void bindNumericArray_doesNotPreservePrecision() {
 
     Struct row =
         execute(
-            Statement.newBuilder("SELECT @v").bind("v").toNumericArray(asList(b1, b2, null)),
+            Statement.newBuilder(selectValueQuery).bind("p1").toNumericArray(asList(b1, b2, null)),
```

Comment:
This test case seem to be disabled for PostgreSQL still. Is that intentional?
The same also seems to be the case for other numeric array tests.

Reply:
missed those, adding it